member-event-guidelines.md

```diff
@@ -335,6 +342,8 @@ organizers must meet the following requirements:
     providing volunteers, our MLH rep will be freed up to help your
     organizing team and hackers with higher priority event support.
 
+    - **Asia-Pacific:** Reservation of a table for the MLH Hardware Lab is not required.
```

Comment:
It's required as of now imo

Reply:
Required only if the hardware lab is available at their event